src/App.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useRef } from 'react';
+import { useSelector } from 'react-redux';
+import { Redirect, Route, Switch } from 'react-router-dom';
+import PrivateRoute from './components/_common/PrivateRoute';
+import { appThemeSelector } from './store/selectors';
+import FullscreenButton from './components/FullscreenButton/FullscreenButton';
+import SignIn from './components/SignIn/SignIn';
+import SignUp from './components/SignUp/SignUp';
+import Main from './components/Main/Main';
```

Comment:
avoid default exports

https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad